compiler/src/dotty/tools/dotc/Compiler.scala

```diff
@@ -95,7 +95,8 @@ class Compiler {
                                        // Note: in this mini-phase block scopes are incorrect. No phases that rely on scopes should be here
            new ElimStaticThis,      // Replace `this` references to static objects by global identifiers
            new Flatten,             // Lift all inner classes to package scope
-           new RestoreScopes),      // Repair scopes rendered invalid by moving definitions in prior phases of the group
+           new RestoreScopes,       // Repair scopes rendered invalid by moving definitions in prior phases of the group
+           new TransformWildcards), // Replace wildcards with default values
```

Comment:
don't put phases in this block... This block is.... strage, se note above.
Better put it in the next block that has backend-specific phases similar to this one.